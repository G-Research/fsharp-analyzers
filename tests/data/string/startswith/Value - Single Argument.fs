let a = "a"
a.StartsWith("a")
