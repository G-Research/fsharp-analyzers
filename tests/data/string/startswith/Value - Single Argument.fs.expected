GRA-STRING-002 | Warning | (2,2 - 2,12) | The usage of String.StartsWith with a single string argument is discouraged. Signal your intention explicitly by calling an overload.
